SU2_CFD/include/numerics_structure.hpp

```diff
@@ -3454,6 +3511,32 @@ class CAvgGradCorrected_Flow : public CNumerics {
 	 * \param[in] val_tauwall_j - Value of the wall shear stress at point j.
 	 */
   void SetTauWall(su2double val_tauwall_i, su2double val_tauwall_j);
+
+  /*!
+   * \brief Initialize the Reynolds Stress Matrix
+   * \param turb_ke turbulent kinetic energy of node
```

Comment:
Very nice to see all the doxy comments 👍 !

Don't forget to add the '[in]' to 'param' here in many of these, or I think it will cause problems with doxygen later. Might want to do a sweep through your PR for this.